src/Symfony/Component/Routing/Tests/Fixtures/Psr4Controllers/SubNamespace/IrrelevantInterface.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Symfony\Component\Routing\Tests\Fixtures\Psr4Controllers\SubNamespace;;
```

Comment:
only one `;` at the end in enough

Reply:
Thank you for reviewing my test fixtures. Most reviewers skip them. ❤️